scene/3d/skeleton_3d.cpp

```diff
@@ -1097,6 +1101,14 @@ void Skeleton3D::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("physical_bones_start_simulation", "bones"), &Skeleton3D::physical_bones_start_simulation_on, DEFVAL(Array()));
 	ClassDB::bind_method(D_METHOD("physical_bones_add_collision_exception", "exception"), &Skeleton3D::physical_bones_add_collision_exception);
 	ClassDB::bind_method(D_METHOD("physical_bones_remove_collision_exception", "exception"), &Skeleton3D::physical_bones_remove_collision_exception);
+
+	ADD_GROUP("Deprecated", "");
+	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "animate_physical_bones"), "set_animate_physical_bones", "get_animate_physical_bones");
+
+	// Note:
+	// In the old Skeleton3D, it really was fired every time the pose was changed, which means it could be fired dozens of times in one frame.
+	// It was very dangerous in terms of performance and potentially breaking the processing order, so for now it should just be an alias of "skeleton_updated".
+	ADD_SIGNAL(MethodInfo("bone_pose_updated"));
```

Comment:
Sorry, my bad. It looks like the signal was called `bone_pose_changed`. It also had an int parameter `bone_idx`.

```suggestion
	ADD_SIGNAL(MethodInfo("bone_pose_changed", PropertyInfo(Variant::INT, "bone_idx")));
```

Reply:
Since there is no compatibility for it, it may need to return an invalid value such as `-1`.

Reply:
:thinking: I see, not sure how disruptive that is compared to removing the signal. If this is going to break users anyway then maybe it's not worth adding it back. I feel like I'm not familiar enough with this area to make this decision.

Reply:
While it is possible to actually make changes to a particular bone only once per frame, it would result in a quite a bit of performance loss as quaternion comparisons would need to be made for all bones; More specifically, the pose of the previous frame should be recorded and compared with the skeleton_updated signal. However, this should be done on the user side, since firing the signal for a particular bone will cause comparisons of other bones that do not need to be detected,

I have written lightly about the reasons for returning `-1` in the documentation, but I think this signal should be removed IMO.